trending_data/src/main/webapp/script.js

```diff
@@ -13,27 +12,9 @@ function initMap() {
     zoom: 8,
```

Comment:
Maybe we could change zoom to the 1 to see all the map

Reply:
Yes, good suggestion